pkg/log/log.go

```diff
@@ -0,0 +1,68 @@
+package log
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Level describes the current log level.
+type Level uint32
+
+const (
+	// DebugLevel log level.
+	DebugLevel Level = Level(logrus.DebugLevel)
+	// InfoLevel log level.
+	InfoLevel Level = Level(logrus.InfoLevel)
+	// ErrorLevel log level.
+	ErrorLevel Level = Level(logrus.ErrorLevel)
+)
+
+type logger struct {
+	level Level
```

Comment:
We can revise the implementation later. For now, this interface helps modifying / adding the call points.